core/base_algo.py

```diff
@@ -371,16 +371,13 @@ def get_remaining_categories(self) -> Set:
         return ALL_CATEGORIES - categories_in_file
 
     def _update_results_csv(self, values_dict) -> None:
-        print('Entered update results csv')  # TODO: Remove this
```

Comment:
maybe we should replace prints with logging?
if you want to maybe do it later then perhaps adding a TODO will suffice.

Reply:
No, I have no idea why this even stayed in our code